src/sdk/http/http-client.test.ts

```diff
@@ -42,6 +42,16 @@ describe('HttpClient', () => {
 			}
 			const url = HttpClient.getUrl('http://localhost:8000/v1/connections', 'http://localhost:9090/', params);
 			expect(url).to.eql('http://localhost:8000/v1/connections?bar=barParameter&baz=bazParameter');
-		})
+		});
+		
+		it('should handle request path that has existing query string', () => {
+			const params = {
+				foo: undefined,
+				bar: 'barParameter',
+				baz: 'bazParameter'
+			}
```

Comment:
We should terminate the const with a semicolon here and in a few more places in this module.

Reply:
It looks like the lint settings need some love. I pushed `fix-lint` to update the lint settings and fix any issues. Once that is merged, I'll rebase this and fix any outstanding issues.